controllers/tab/settings/siteSetup/form/SiteSetupForm.inc.php

```diff
@@ -75,12 +80,24 @@ function fetch($request, $params = null) {
 		$application = Application::getApplication();
 		$templateMgr->assign('availableMetricTypes', $application->getMetricTypes(true));
 
-		$themePlugins = PluginRegistry::loadCategory('themes');
-		$themePluginOptions = array();
+		$themePlugins = PluginRegistry::getPlugins('themes');
+		if (is_null($themePlugins)) {
+			$themePlugins = PluginRegistry::loadCategory('themes');
```

Comment:
Is this load necessary? It should've been loaded already in the constructor, I think.